osu.Game/Overlays/Mods/ModCustomisationHeader.cs

```diff
@@ -109,15 +111,40 @@ protected override void LoadComplete()
             ExpandedState.BindValueChanged(v =>
             {
                 icon.ScaleTo(v.NewValue > ModCustomisationPanelState.Collapsed ? new Vector2(1, -1) : Vector2.One, 300, Easing.OutQuint);
+
+                switch (v.NewValue)
+                {
+                    case ModCustomisationPanelState.Collapsed:
+                        background.FadeColour(colourProvider.Dark3, 500, Easing.OutQuint);
+                        break;
+
+                    case ModCustomisationPanelState.Expanded:
+                    case ModCustomisationPanelState.ExpandedByMod:
+                        background.FadeColour(colourProvider.Light4, 500, Easing.OutQuint);
+                        break;
+                }
             }, true);
         }
 
         protected override bool OnHover(HoverEvent e)
         {
-            if (Enabled.Value && panel.ExpandedState.Value == ModCustomisationPanelState.Collapsed)
+            if (!Enabled.Value)
+                return base.OnHover(e);
+
+            if (panel.ExpandedState.Value == ModCustomisationPanelState.Collapsed)
                 panel.ExpandedState.Value = ModCustomisationPanelState.Expanded;
 
+            hoverBackground.FadeIn(200);
```

Comment:
This hover effect is too bright when the panel is expanded. Leads to some pretty awful contrast when compared to the dropdown title text.

![image](https://github.com/user-attachments/assets/34f6c107-f79a-46c0-a980-509f0536bc56)

Reply:
Adjusted down a bit, probably fine now.